### DISCUSSION TITLE
how do I include the output of a bash command into my markup file?

### DISCUSSION THREAD

**Discussion by wf5w** [gollum/gollum#1868] [Q&A] [answered]
Hey y'all:

I want to include the output of a bash command into my preview

1. can I do this?
2. what does the markup look like?
3. where does that bash script need to be located?

Thanks.
Jerry

**Comment by dometto** [comment]
There's currently no support for this, but this functionality could be added through a [Custom Macro](https://github.com/gollum/gollum/wiki#custom-macros) that evaluates either a file or a string passed to the Macro (see e.g. [here](https://stackoverflow.com/questions/2232/how-to-call-shell-commands-from-ruby)). Because of safety concerns we will not include such functionality in gollum, but we'd be happy to help out if you want to try your hand at such a Macro yourself.

**Comment by wf5w** [comment]
thanks. this is my personal wiki.
And I wanted to include the current date

thanks.

**Comment by dometto** [comment, marked as answer]
You probably want to use a Template Filter for that: https://github.com/gollum/gollum/wiki#template-filter
